packages/@react-stately/table/src/useTableState.ts

```diff
@@ -26,7 +26,11 @@ export interface TableState<T> extends GridState<T, ITableCollection<T>> {
   /** The current sorted column and direction. */
   sortDescriptor: SortDescriptor,
   /** Calls the provided onSortChange handler with the provided column key and sort direction. */
-  sort(columnKey: Key, direction?: 'ascending' | 'descending'): void
+  sort(columnKey: Key, direction?: 'ascending' | 'descending'): void,
+  /** Whether keyboard navigation is disabled, such as when you need to use the arrow keys to interact with an internal component. */
+  isKeyboardNavigationDisabled: boolean,
+  /** Set whether keyboard navigation is disabled, such as when you need to use the arrow keys to interact with an internal component. */
+  setIsKeyboardNavigationDisabled: (val: boolean) => void
```

Comment:
Usually the setter doesn't contain "is"

```suggestion
  setKeyboardNavigationDisabled: (val: boolean) => void
```